vision/cloud-client/product_search/product_management.py

```diff
@@ -181,6 +183,39 @@ def delete_product(project_id, location, product_id):
 # [END vision_product_search_delete_product]
 
 
+# [START vision_product_search_purge_orphan_products]
+def purge_orphan_products(project_id, location, force):
+    """Delete all products not in any product sets.
+    Args:
+        project_id: Id of the project.
+        location: A compute region name.
+    """
+    client = vision.ProductSearchClient()
+
+    parent = client.location_path(
+        project=project_id, location=location)
+
+    # The purge operation is async.
+    operation = client.purge_products(
+        parent=parent,
+        delete_orphan_products=True,
+        # The operation is irreversible and removes multiple products.
+        # The user is required to pass in force=True to actually perform the
+        # purge.
+        # If force is not set to True, the service raises an exception.
+        force=force)
+
+    # TODO: replace with operation.result(timeout=60)

```

Comment:
Java there is already method for this purpose, java--> ` response.get(60, TimeUnit.SECONDS);` However, C# --> `operation.Result.PollUntilCompleted();` which doesn't contain timeout option.   Do we really have to include this 60 sec timeout for every language?

Reply:
specifying the timeout is not required, and does not need to be consistent across languages.  if possible each language should handle this in the language-specific conventional approach.